### ISSUE TITLE
measured values column is not extracted

### ISSUE THREAD

**Issue by floydluo** [bertrandmartel/tableau-scraping#9]
Hi @bertrandmartel, I met a new problem when using TS to scrape the data from the tableau of New York State. Its URL is: https://covid19vaccine.health.ny.gov/vaccine-demographic-data

The original data look like this: 

![image](https://user-images.githubusercontent.com/18824134/113928222-7b1bd700-97bc-11eb-82f3-1d41d2a4b05f.png)

I tried the following code to scrape the data: 

```python
from tableauscraper import TableauScraper as TS

url = "https://covid19tracker.health.ny.gov/views/Race_Ethnicity_Public/RacebyCounty"

ts = TS()
ts.loads(url)

workbook = ts.getWorkbook()

parameters = workbook.getParameters()
print(parameters)

ts = TS()
ts.loads(url)

# set parameters column / value
workbook = workbook.setParameter('Show Value as', "Number")
# display worksheets
workbook.getWorksheet('Race').data
```

But what I get is this:

![image](https://user-images.githubusercontent.com/18824134/113928384-b7e7ce00-97bc-11eb-9aaf-4e9fb93d486e.png)

So the detailed values are replaced with `%all%`. I don't understand why. Do you have any suggestions?

Thanks very much!

**Comment by bertrandmartel** [comment]
@floydluo  sorry for late response, it seems `%all%` is normal here, it means that this rows concerns all region/counties eg  as if this value were empty in a datatable. I guess the issue is more that the `Measure Names-value` column is not returning data but something like `[federated.0nvcif80f24s4c147rz6d1mpve6p (copy)].[usr:Calculation_626563313086001155:qk]` which looks like calculated fields if I'm not mistaken. 
Maybe the data is in other sheets ?

**Comment by bertrandmartel** [comment]
@floydluo it seems the `Measure Names` is present in the dataset. There must be a mecanism to get the calculated field based on `Measure Names`

**Comment by bertrandmartel** [comment]
I think I can extract them by getting the sort parameters in`sortIndicators.sortIndicatorRegions[...].sortIndicatorItems[...].sortIndicatorMenu.commandItems[...]` under `vizdata` which has the following model:

```javascript
{
    'commandsType': 'subcommands',
    'name': 'Field',
    'telemetryId': '2335452376',
    'commands': {
        'commandItems': [{
            'commandsType': 'item',
            'name': 'AGG(Race Total Population Aged 15+)',
            'telemetryId': '0',
            'command': 'tabdoc:sort aggregation="user" global-field-name="[federated.0nvcif80f24s4c147rz6d1mpve6p (copy)].[none:pat_region_desc:nk]" set-default="false" sort-by="field" sort-measure-name="[federated.0nvcif80f24s4c147rz6d1mpve6p (copy)].[usr:Calculation_1568378586972782592:qk]" sort-order="desc"'
        }, {
            'commandsType': 'item',
            'name': 'AGG(Vaccination by Race)',
            'telemetryId': '0',
            'command': 'tabdoc:sort aggregation="user" global-field-name="[federated.0nvcif80f24s4c147rz6d1mpve6p (copy)].[none:pat_region_desc:nk]" set-default="false" sort-by="field" sort-measure-name="[federated.0nvcif80f24s4c147rz6d1mpve6p (copy)].[usr:Calculation_626563313086001155:qk]" sort-order="desc"'
        }]
    }
}
```

I would add 2 columns computed with `.name` and `global-field-name` under `command` as there can be many commands. In your request response, there are 4 commands, 2 commands for regions / 2 for zip county.

For the data I would get `global-field-name` and search the dataset for this `global-field-name` value. (its is a dataset not mapping a worksheet, this is why it doesn't appear when you perform `getWorksheets()`)

**Comment by bertrandmartel** [comment]
I've found out there is a column missing that holds the values, this columns is marked as type `real` whereas it seems it needs to read the data from the `cstring` field

**Comment by bertrandmartel** [comment]
It is fixed in v0.1.8:

```python
from tableauscraper import TableauScraper as TS

url = "https://covid19tracker.health.ny.gov/views/Race_Ethnicity_Public/RacebyCounty"

ts = TS()
ts.loads(url)

workbook = ts.getWorkbook()

parameters = workbook.getParameters()
print(parameters)

ts = TS()
ts.loads(url)

# set parameters column / value
workbook = workbook.setParameter('Show Value as', "Number")
# display worksheets
print(workbook.getWorksheet('Race').data)
```

**Comment by floydluo** [comment]
@bertrandmartel That's great! Thank you.
